csharp/src/Google.Protobuf/Collections/RepeatedField.cs

```diff
@@ -51,6 +51,7 @@ public sealed class RepeatedField<T> : IList<T>, IList, IDeepCloneable<RepeatedF
         , IReadOnlyList<T>
 #endif
     {
+        private static readonly EqualityComparer<T> EqualityComparer = ProtobufEqualityComparers.GetEqualityComparer<T>();
```

Comment:
I didn't see the whole file. But can't we use the name comparer here and remove line 438 and 458?

Reply:
The point of those lines is to copy the static field value into a local variable *once* rather than on each iteration.